jobs/integr8ly/browser-based-testsuite-pipeline.yaml

```diff
@@ -34,31 +34,47 @@
           description: 'Evals account email address used to access the available consoles'
     dsl: |
         timeout(60) {
+
             node('cirhos_rhel7') {  
+
+                Boolean publishTestResults = true
+
                 stage('Clone the testsuite') {
                     dir('integreatly-qe') {
-                            git branch: BRANCH, url: REPOSITORY
+                        git branch: BRANCH, url: REPOSITORY
                     } 
                 }
                 
                 docker.image('selenium/standalone-chrome:3.14.0-krypton').withRun('--name chrome_selenium -d -p 4444:4444 -v /dev/shm:/dev/shm -v "$PWD":"$PWD" -e SE_OPTS="-timeout 3600"') { c ->
                     stage('Test') {
                         dir('integreatly-qe/tests') {
-                            sh """
-                            # Disable starting Selenium, Docker is used instead
-                            sed -i 's/"start_process" : true,/"start_process" : false,/g' nightwatch.json
-                            npm install
-                            npm test
-                            """
+                            try {
+                                sh """
+                                # Disable starting Selenium, Docker is used instead
+                                sed -i 's/"start_process" : true,/"start_process" : false,/g' nightwatch.json
+                                npm install
+                                npm test
+                                """   
+                            } catch (Exception e) {
+                                String reportsDirEntriesCount = sh returnStdout: true, script: 'ls ui-testsuite/reports | wc -l'
+                                if (reportsDirEntriesCount.trim() == '0') {
+                                    currentBuild.result = 'FAILURE'
+                                    publishTestResults = false
+                                }
+                            }
                         } 
                     }
                 }
 
                 stage('Publish test results') {
-                    dir('integreatly-qe/tests/ui-testsuite/reports') {
-                        archiveArtifacts '**/*.xml'                  
-                        junit allowEmptyResults:true, testResults: '**/*.xml'
-                    } 
+                    if (publishTestResults) {
```

Comment:
Do we still need this now? We have `allowEmptyResults:true` so I would guess that we don't need to handle empty reports folder.

Reply:
you are right. Removed it